src/Nethermind/Nethermind.TxPool/AddTxResult.cs

```diff
@@ -71,6 +76,11 @@ public enum AddTxResult
         /// </summary>
         Invalid,
         
+        /// <summary>
+        /// Permission denied for this tx type.
+        /// </summary>
+        PermissionDeniedForThisTxType,
```

Comment:
just PermissionDenied

Reply:
Also I don't love the fact that POSDAO based specific result now got into here. Maybe we could make AddTxResult a struct tha would have:
1) Number
2) Code
3) Optional custom message?

Then other modules when extending TxPool could return their won results.